ci_framework/roles/edpm_build_images/tasks/main.yml

```diff
@@ -36,6 +36,10 @@
   retries: 60
   delay: 10
 
+- name: Add certs to the image
+  ansible.builtin.import_tasks: add_cert.yml
+  when: cifmw_edpm_build_images_cert_install_command is defined
```

Comment:
Minor nit, move when condition 1 line up after name as that's the convention we are using in other places in the repo